com.unity.netcode.gameobjects/Runtime/SceneManagement/SceneEventData.cs

```diff
@@ -268,7 +268,8 @@ internal void AddSpawnedNetworkObjects()
         internal void AddDespawnedInSceneNetworkObjects()
         {
             m_DespawnedInSceneObjectsSync.Clear();
-            var inSceneNetworkObjects = UnityEngine.Object.FindObjectsOfType<NetworkObject>().Where((c) => c.NetworkManager == m_NetworkManager);
+            // Find all active and non-active in-scene placed NetworkObjects
+            var inSceneNetworkObjects = UnityEngine.Object.FindObjectsOfType<NetworkObject>(true).Where((c) => c.NetworkManager == m_NetworkManager);
```

Comment:
nit-pick. Can we have
`var inSceneNetworkObjects = UnityEngine.Object.FindObjectsOfType<NetworkObject>(includeInactive: true).Where((c) => c.NetworkManager == m_NetworkManager);` ? Otherwise, it is especially hard to parse what changed in this PR. :-)